Libplanet.Net.Tests/Consensus/Context/ContextProposerTest.cs

```diff
@@ -281,29 +279,10 @@ void CheckVote(object? observer, ConsensusMsg? message)
                 }
             }
 
-            // Bypass StartAsync() to avoid StartRound(0).
-            _ = context.MessageConsumerTask(default);
-            _ = context.MutationConsumerTask(default);
-
-            var invalidBlock = new BlockContent<DumbAction>(
-                new BlockMetadata(
-                    protocolVersion: BlockMetadata.CurrentProtocolVersion,
-                    index: blockChain.Tip.Index + 1,
-                    timestamp: blockChain.Tip.Timestamp.Subtract(TimeSpan.FromSeconds(1)),
-                    miner: fx.Miner.PublicKey.ToAddress(),
-                    publicKey: fx.Miner.PublicKey,
-                    previousHash: blockChain.Tip.Hash,
-                    txHash: null,
-                    lastCommit: null))
-                .Propose().Evaluate(fx.Miner, blockChain);
-
-            context.ProduceMessage(
-                TestUtils.CreateConsensusPropose(
-                    invalidBlock, TestUtils.Peer1Priv));
-
```

Comment:
Can we have a test case where the node is not a proposer and receives an invalid proposal?

Reply:
See 6a98b6049090c9c40425f7b1df5b49cadf14e6b0.